HelpSource/Classes/DetectSilence.schelp

```diff
@@ -77,3 +78,17 @@ Task({
 )
 
 ::
+
+section::DetectSilence and multiple inputs
+
+Be careful feeding multiple inputs into DetectSilence, since multichannel expansion can lead to confusing behavior. This code
+
+code::DetectSilence.ar([left, right] + Impulse.ar(0), doneAction: 2)::
```

Comment:
Did you intend this to appear as runnable code? That only happens if `code::` and the closing `::` are on individual lines:
```
code::
DetectSilence...
::
```